README.md

```diff
@@ -395,3 +395,12 @@ The token generated by `rake generate_token` should be passed along in the `Auth
 API requests that do not supply a valid token for the target server will be rejected as Unauthorized.
 
 At present, all tokens grant the same (full) access to the read/update API.
+
+## Cron check-ins
+Some cron jobs (configured via the whenever gem) are integrated with Honeybadger check-ins. These cron jobs will check-in with HB (via a curl request to an HB endpoint) whenever run. If a cron job does not check-in as expected, HB will alert.
+
+Cron check-ins are configured in the following locations:
+1. `config/schedule.rb`: This specifies which cron jobs check-in and what setting keys to use for the checkin key. See this file for more details.
+2. `config/settings.yml`: Stubs out a check-in key for each cron job. Since we may not want to have a check-in for all environments, this stub key will be used and produce a null check-in.
+3. `config/settings/production.yml` in shared_configs: This contains the actual check-in keys.
+4. HB notification page: Check-ins are configured per project in HB. To configure a check-in, the cron schedule will be needed, which can be found with `bundle exec whenever`. After a check-in is created, the check-in key will be available. (If the URL is `https://api.honeybadger.io/v1/check_in/rkIdpB` then the check-in key will be `rkIdp`).
```

Comment:
@justinlittman The end of the URL is `rkIdpB` and the check-in key is `rkIdp` (without the trailing `B`)... is that intended?

Reply:
I submitted a PR to fix this since I didn't catch it during review.

https://github.com/sul-dlss/preservation_catalog/pull/2100